### PR TITLE
Resize organ contents after small

### PR DESCRIPTION
**Pull request by gausie**
Because this is the first liberateKing test, it caused some leakage that meant I needed to fix up some other tests to get this passing. I've kept those improvements here, so this PR is bigger than it should otherwise be!

I made a bunch of changes so that the leakage wouldn't happen, but mafia is just not set up to re-initiate the ConcoctionPool. When I stepped back and thought about it I realised that the test I was trying to make work was actually asserting something false (sort of)

### CODE REVIEW

File: test/net/sourceforge/kolmafia/KoLCharacterTest.java [hunk 1]
```diff
@@ -548,4 +548,16 @@ public void oldQuestDoesNotGiveExperience() {
       }
     }
   }
+
+  @Nested
+  class Liberation {
+    @Test
+    void resizesOrganContentsAfterSmallPath() {
+      try (var cleanups = new Cleanups(withPath(Path.SMALL), withInebriety(1), withFullness(1))) {
```
Comment (Alium58) [inline test/net/sourceforge/kolmafia/KoLCharacterTest.java:556]: Small can have up to 2 fullness used, which updates to 20 after liberation. Might be interesting to test this as it is above the typical fullness limit

Comment (gausie) [reply test/net/sourceforge/kolmafia/KoLCharacterTest.java:556]: Sure. I am just multiplying the number by 10 ;)

**Comment by gausie** [comment]
I do not understand the failing test here...

**Comment by gausie** [comment]
I understand the failing test now, turns out liberating the king leaks all sorts of things everywhere in the test environment

**Review by midgleyc — APPROVED** [review]
Mostly testing changes ;)

**Pull request merged**